bpf/lib/common.h

```diff
@@ -486,6 +486,18 @@ struct egress_gw_policy_entry {
 	__u32 gateway_ip;
 };
 
+struct egress_gw_policy_key6 {
+	struct bpf_lpm_trie_key lpm_key;
+	union v6addr saddr;
+	union v6addr daddr;
+};
+
+struct egress_gw_policy_entry6 {
+	__u32 gateway_ip;
+	union v6addr egress_ip;
+	__u32 reserved[3]; /* reserved for future extension, e.g. v6 gateway_ip */
```

Comment:
```suggestion
	union v6addr egress_ip;
	__u32 gateway_ip;
	__u32 reserved[3]; /* reserved for future extension, e.g. v6 gateway_ip */
```

This should help with struct-internal alignment.